pages/data.py

```diff
@@ -77,8 +77,23 @@ def render_content(tab, store_uuids, store_trips, store_demographics, store_traj
         has_perm = perm_utils.has_permission('data_trips')
     elif tab == 'tab-demographics-datatable':
         data = store_demographics["data"]
-        columns = list(data[0].keys())
         has_perm = perm_utils.has_permission('data_demographics')
+        # if only one survey is available, process it without creating a subtab
+        if len(data) == 1: 
+            # here data is a dictionary 
+            data = next(iter(data.values()))
```

Comment:
@achasmita I had commented on this in:
https://github.com/e-mission/op-admin-dashboard/pull/89#discussion_r1388628743

you removed it in 
https://github.com/e-mission/op-admin-dashboard/pull/89/commits/53dafd6af1a2f4d36f0cabd49b3418fc24973539

but now that I check the code, this is still here.

Please fix.